kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -111,8 +111,14 @@ final class ProducerMetrics {
   private final String responseSensorName;
   private final String recordErrorSensorName;
   private final String requestLatencySensorName;
+  private final Map<String, String> metricsTags;
```

Comment:
I know it's cumbersome to pass this down to `createMetrics` all the way to `getMetricName`, but this variable is not actually used after the constructor exits, so it shouldn't really be available. Please, pass it down instead. :)

Reply:
I'm not entirely sure I agree - metricsTags is a property of the ProduerMetrics in the same way that the Name fields are. I've typed up the change now though, so take a look.